windows/RNFileViewer/RNFileViewer.csproj

```diff
@@ -108,23 +108,28 @@
   <ItemGroup>
     <Compile Include="RNFileViewerModule.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="RNFileViewerPackage.cs" />
+    <Compile Include="ReactPackageProvider.cs" />
     <EmbeddedResource Include="Properties\RNFileViewer.rd.xml" />
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
-      <Version>6.0.8</Version>
+      <Version>6.2.10</Version>
     </PackageReference>
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\..\react-native-windows\ReactWindows\ReactNative\ReactNative.csproj">
-      <Project>{C7673AD5-E3AA-468C-A5FD-FA38154E205C}</Project>
-      <Name>ReactNative</Name>
+    <ProjectReference Include="..\..\..\react-native-windows\Microsoft.ReactNative\Microsoft.ReactNative.vcxproj">
+      <Project>{f7d32bd0-2749-483e-9a0d-1635ef7e3136}</Project>
+      <Name>Microsoft.ReactNative</Name>
+      <Private>False</Private>
     </ProjectReference>
   </ItemGroup>
+  <Import Project="..\..\..\react-native-windows\Microsoft.ReactNative.SharedManaged\Microsoft.ReactNative.SharedManaged.projitems" Label="Shared" />
```

Comment:
This line has been deprecated and should be removed starting from RN63. I've tested and actually I wasn't able to import the project.